pytensor/link/pytorch/dispatch/scalar.py

```diff
@@ -38,3 +37,24 @@ def pytorch_func(*args):
             )
 
     return pytorch_func
+
+
+@pytorch_funcify.register(ScalarLoop)
+def pytorch_funicify_ScalarLoop(op, node, **kwargs):
+    update = pytorch_funcify(op.fgraph)
```

Comment:
I didn't `torch.compile` this - not sure if we need to. I think the torch.compile in the linker will do fine (and hopefully get a bit of a loop reduction for us)